event.go

```diff
@@ -828,3 +830,21 @@ func (e *Event) MACAddr(key string, ha net.HardwareAddr) *Event {
 	e.buf = enc.AppendMACAddr(enc.AppendKey(e.buf, key), ha)
 	return e
 }
+
+// GetFields returns the JSON decoded fields of the event. Please note that the fields might not be mapped to their original type as JSON decoder maps numbers to float64, etc.
+func (e *Event) GetFields() (map[string]interface{}, error) {
+	if e == nil {
+		return nil, nil
+	}
+
+	eventFields := make(map[string]interface{})
+	buffer := bytes.TrimSpace(e.buf)
+	// If `Msg()` was not called the buffer will be missing the closing curly brace
+	if !bytes.HasSuffix(buffer, []byte("}")) {
```

Comment:
```suggestion
	if !bytes.HasSuffix(buffer, '}')) {
```

Reply:
OK tried this and it doesn't seem to work as `HasSuffix` accepts only a `[]byte`